gdx/src/com/badlogic/gdx/math/MathUtils.java

```diff
@@ -182,6 +182,24 @@ static public int randomSign () {
 		return 1 | (random.nextInt() >> 31);
 	}
 
+	/** Returns a random point inside the unit circle */
+	static public Vector2 randomUnitCirclePoint (Vector2 vector) {

```

Comment:
This is actually a random point in a disk, not a random point in a circle.

https://en.wikipedia.org/wiki/Disk_(mathematics) Disk is the area bounded by a circle. (circle is only the boundaries, disk is everything inside the circle)